assets/training/finetune_acft_hf_nlp/components_v2/pipeline_components/text_generation/spec.yaml

```diff
@@ -141,6 +141,7 @@ jobs:
       train_file_path: '${{parent.inputs.train_file_path}}'
       validation_file_path: '${{parent.inputs.validation_file_path}}'
       system_properties: '${{parent.inputs.system_properties}}'
+      user_column_names: '${{parent.inputs.text_key}},${{parent.inputs.ground_truth_key}}'
```

Comment:
Why this change is needed?

Reply:
Accidentally got removed in previous PR, readding it now